packages/twenty-front/src/modules/analytics/hooks/useAnalyticsGraphDataState.ts

```diff
@@ -0,0 +1,17 @@
+import { webhookAnalyticsGraphDataState } from '@/analytics/states/webhookAnalyticsGraphDataState';
+
+import { mapWebhookAnalyticsResultToNivoLineInput } from '@/analytics/utils/mapWebhookAnalyticsResultToNivoLineInput';
+
+export const useAnalyticsGraphDataState = (endpointName: string) => {
+  switch (endpointName) {
+    case 'getWebhookAnalytics':
+      return {
+        analyticsState: webhookAnalyticsGraphDataState,
```

Comment:
It feels like what you want to use here might be componentState? Look for `createComponentState` in the code :)